page/sass/footer.scss

```diff
@@ -13,13 +21,17 @@
             position:static;
             width: 260px;
             height: 260px;
-            transform: rotate(200deg) translateY(-20px);
+            transform: rotate(134deg) translateY(-36px);
         }
     }
 }
-
+.border-top{
+    border-top: 1px $off-white solid;
+    padding-top: 20px;
+}
 .footer{
-    padding-top: $padding-between-sections;
+    padding-top: $padding-between-sections;   
+    border:0 !important;  
```

Comment:
Why is important necessary, is there something overriding this even after stating style normally?